aquarium.py

```diff
@@ -69,7 +69,7 @@ def build_src(src_dir, version, git_url):
         subprocess.check_call(["git", "-C", f"{SRC_DIR}", "clone", git_url])
 
     subprocess.check_call(["git", "-C", f"{src_dir}", "fetch", "origin", version])
-    subprocess.check_call(["git", "-C", f"{src_dir}", "checkout", f"origin/{version}"])
+    subprocess.check_call(["git", "-C", f"{src_dir}", "checkout", f"tags/{version}"])
```

Comment:
Hmm this will cause issue if we update it to branches. How about dropping it entirely?
```suggestion
    subprocess.check_call(["git", "-C", f"{src_dir}", "checkout", f"{version}"])
```